cmd/ops_agent_windows/run_windows.go

```diff
@@ -70,8 +71,9 @@ func (s *service) Execute(args []string, r <-chan svc.ChangeRequest, changes cha
 func (s *service) parseFlags(args []string) error {
 	s.log.Info(1, fmt.Sprintf("args: %#v", args))
 	var fs flag.FlagSet
-	fs.StringVar(&s.inFile, "in", "", "input filename")
-	fs.StringVar(&s.outDirectory, "out", "", "output directory")
+	fs.StringVar(&s.userConf, "in", "", "path to read the user specified agent config")
```

Comment:
`path to the user specified agent config`

Reply:
Done